src/DynamoCoreWpf/Controls/StandardPanel.xaml.cs

```diff
@@ -47,7 +47,18 @@ private void OnHeaderButtonClick(object sender, RoutedEventArgs e)
             e.Handled = true;
         }
 
-        private void OnMouseLeftButtonDown(object sender, MouseButtonEventArgs e)
+        private void OnMouseMove(object sender, MouseEventArgs e)
+        {
+            if (e.LeftButton != MouseButtonState.Pressed)
+                return;
+
+            var listBoxItem = sender as ListBoxItem;
+            var searchElementVM = listBoxItem.DataContext as NodeSearchElementViewModel;
+
```

Comment:
@aosyatnik We should make sure to check these two properties for null here.  If they are null, we should return prematurely.